graphql-cli/build.gradle

```diff
@@ -112,6 +113,9 @@ test {
     systemProperty "ballerina.home", bDistribution
 
     useTestNG() {
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            excludeGroups('invalid_permission')
```

Comment:
Better to define these as a separate variable and use it. Otherwise, this will be hard to find.

Reply:
Fixed in https://github.com/ballerina-platform/graphql-tools/pull/122/commits/464a5a604753e5c4b5440cdd4e783bfb917643a7